src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
```

Comment:
Simplify the conditional structure by removing the 'else' keyword. After the 'if' block that checks for a rental period of 7 or more days, you can directly check if the days are 3 or more without 'else' since if the first condition is true, the function will return early.